### PR TITLE
Date selection in search menu missing higher Z-index value

### PR DESCRIPTION
**Pull request by rijaspk**
Date selection in search menu missing a higher Z-index value
Issue #14248

Signed-off-by: Rijas Ahammed PK <rijaspk5195@gmail.com>

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [ ] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

### CODE REVIEW

File: themes/original/jquery/jquery-ui.css [hunk 1]
```diff
@@ -403,6 +403,7 @@ button.ui-button::-moz-focus-inner {
 	width: 17em;
 	padding: .2em .2em 0;
 	display: none;
+       z-index: 99 !important;
```
Comment (mauriciofauth) [inline themes/original/jquery/jquery-ui.css]: Do you have any other way to solve it without using `!important`?

**Review by MauricioFauth — COMMENTED** [review]
@rijaspk Thanks for your contribution.

File: themes/original/jquery/jquery-ui.css (hunk 1 above)
Comment (rijaspk) [reply themes/original/jquery/jquery-ui.css]: Hi @mauriciofauth, Date picker UI used Inline style and unfortunately I couldn't find the file where it is written. Can you help me find that file

File: themes/original/jquery/jquery-ui.css [hunk 2]
```diff
@@ -170,7 +170,7 @@
 }
 .ui-selectable-helper {
 	position: absolute;
-	z-index: 100;
+       z-index: 100;
```
Comment (saulotoledo) [inline themes/original/jquery/jquery-ui.css]: This file uses tab as indent, please fix it.

File: themes/original/jquery/jquery-ui.css (hunk 1 above)
Comment (saulotoledo) [reply themes/original/jquery/jquery-ui.css]: Hi everyone. I am new here, but I would like to head the opinion of an old repository reviewer about. This is a JQuery UI file, so I think we should do these changes in another place, not here. Are we really allowed do change these files?

**Review by saulotoledo — CHANGES_REQUESTED** [review]

File: js/server_databases.js [hunk 3]
```diff
@@ -101,7 +101,7 @@ AJAX.registerOnload('server_databases.js', function () {
 
         // TODO Remove this section when all browsers support HTML5 "required" property
         var newDbNameInput = $form.find('input[name=new_db]');
-        if (newDbNameInput.val() === '') {
+        if (newDbNameInput.val() == '') {
```
Comment (mauriciofauth) [inline js/server_databases.js]: This change does not appear to be related to this pull request.
If this is fixing another issue, it is better to open a new pull request with this change.

File: themes/original/jquery/jquery-ui.css (hunk 1 above)
Comment (mauriciofauth) [reply themes/original/jquery/jquery-ui.css]: Hi, @saulotoledo. You're right, this is a jQuery UI file. Although this file already contains some changes, I agree that we should not edit this file, and that those changes should go elsewhere.

**Pull request closed without merging**